site/snippets/sidebar/menu-grouped.php

```diff
@@ -0,0 +1,34 @@
+<?php
+extract([
+	'open' => $open ?? false
+])
+?>
+<?php foreach ($menu->group('category') as $category => $items): ?>
```

Comment:
Any chance to combine `menu` and `menu-grouped` snippets to have less redundancy?